Draggable.tsx

```diff
@@ -45,6 +45,7 @@ interface IProps {
     onPressIn?: (event: GestureResponderEvent) => void;
     onPressOut?: (event: GestureResponderEvent) => void;
     onRelease?: (event: GestureResponderEvent, wasDragging: boolean) => void;
+    onReverse?: () => {x: number, y: number},
```

Comment:
If we're keeping `Draggable.tsx` we should update the other places changed

Reply:
removed it